### ISSUE TITLE
[EN][The Obsidian Sanctum][Raid] Sarhtarion enrage

### ISSUE THREAD

**Issue by captjvmoore** [Atlantiss/BugTracker#1771]
We engaged Sartharion http://www.wowhead.com/npc=28860/sartharion with all 3 Twilight Drakes still up, which is equal to Hard Mode. When his health drops to about 2500k ~ 3000k, he enrages and wipes the raid. Instead, the raid should have a 15 min. enrage timer window before that happens.

**Comment by Qwers** [comment]
Firstly read this http://www.wowhead.com/spell=61254/will-of-sartharion and then read tactic.

**Comment by captjvmoore** [comment]
What I'm talking about is http://www.wowhead.com/spell=61632/berserk Qwers...

**Comment by Alandiel** [comment]
He enrages when the drakes are still alive and his hp is less than 50%. You need to kill 3 drakes and then kill him.

**Comment by Glare** [comment]
"However, should Sartharion reach 35% with any of the Twilight Drakes alive, he will cast this on them, leading to death."

Engage him while 3 drakes are still alive BUT wait for drakes to fly down, kill them first and then kill Sartharion.

**Comment by captjvmoore** [comment]
Let me explain what we saw.
We tried all of what you suggested. In both occasions, the outcome was the same. When the boss reached 2500k ~ 3000k he enraged with this http://www.wowhead.com/spell=61632/berserk and started doing 50k melee cleaves to the tank, very fast. Of course, healers can't top this, so the tank died, then the boss turned around and hit the entire raid for 130k with this http://www.wowhead.com/spell=56908/flame-breath. The other tank, the one that was tanking the Twilight Drakes was still alive and taking little-to-no damage.
We tried killing the drakes, as the tactics suggest. The above happened. Then we tried the zerg tactic - same thing happened.
